stan/math/rev/prob/std_normal_log_qf.hpp

```diff
@@ -19,16 +19,18 @@ namespace math {
  */
 template <typename T, require_stan_scalar_or_eigen_t<T>* = nullptr>
 inline auto std_normal_log_qf(const var_value<T>& log_p) {
-  return make_callback_var(
-      std_normal_log_qf(log_p.val()), [log_p](auto& vi) mutable {
-        auto vi_array = as_array_or_scalar(vi.val());
-        auto vi_sign = sign(as_array_or_scalar(vi.adj()));
-
-        const auto& deriv = as_array_or_scalar(log_p).val()
-                            + log(as_array_or_scalar(vi.adj()) * vi_sign)
-                            - NEG_LOG_SQRT_TWO_PI + 0.5 * square(vi_array);
-        as_array_or_scalar(log_p).adj() += vi_sign * exp(deriv);
-      });
+  const auto& arena_rtn = to_arena(std_normal_log_qf(log_p.val()));
```

Comment:
No need for `const auto&` here since to_arena is going to have a new matrix passed into it

```suggestion
  auto arena_rtn = to_arena(std_normal_log_qf(log_p.val()));
```

Reply:
Oh good catch thanks!